### DISCUSSION TITLE
Project beta: support `assignee:@me` filter

### DISCUSSION THREAD

**Discussion by olafurpg** [community/community#5769] [Projects and Issues] [answered]
The filter `assignee:@me` returns empty tasks in the project beta. I expected it to return tasks that are assigned to the person who is currently logged in, similar to how `assignee:@me` works as a filter when searching for issues. This feature would be helpful for my team to have a single view/URL where everybody can see the issues that are assigned to them.

**Comment by erikr** [comment]
Yes, seconded! This would be super helpful.

**Comment by massung** [comment]
Absolutely, especially since the project boards are limited to 10 tabs. I'd like to have general tabs for "My PRs" and "My Open Issues" that each person on the team can go to and see quickly what they need to be working on.

**Comment by faperreault** [comment]
Came here to propose this. We need this to use projects with a lot of team members.

**Comment by parisk** [comment]
Second that. It's super helpful to get a personal view of what's on your plate in a given project.

**Comment by pnegri** [comment]
Super helpful!

**Comment by azenMatt** [comment, marked as answer]
Hi @olafurpg - we just implemented this 😄 

You should now be able to add the `@me` filter to the assignee field which will always map to the current user

![image](https://user-images.githubusercontent.com/7584089/154979467-4fdea73b-4159-4ebc-b102-aadbb373e113.png)

**Reply by olafurpg** [reply]
Wow, thank you! I just tried this out and it works great 👍

**Reply by Marychoffel22** [reply]
Okay 🤝

**Reply by joelanman** [reply]
not sure if I'm doing it wrong, as my Projects view doesnt look like this, but this filter doesnt work for me, (assignee:joelanman does)

<img width="1357" alt="screenshot of Projects view, assignee:@me filter with no cards showing" src="https://user-images.githubusercontent.com/1132904/173800343-6bd4c18a-c889-4185-9b4b-5cba9e5a4b67.png">

**Reply by azenMatt** [reply]
Hi @joelanman - are you on the on-prem version of GitHub? 

The UI you shared is the old version of projects, this feature only works on the new 'beta' version of projects 

This hasn't been released to on-prem yet, however, if you are on the cloud version of GitHub the projects tab should nudge you towards the beta version as the first option

**Reply by joelanman** [reply]
@azenMatt ah thank you! I can see beta now, but it seems like my project is in the old version, is there any way to change it to the beta version?

**Reply by azenMatt** [reply]
Your timing could not be better 😄 we just released a new migration script you can turn on and try out - https://github.blog/changelog/2022-06-16-the-new-github-issues-june-16th-update/
